### PR TITLE
Feat/seedlot feedback page

### PR DESCRIPTION
**Pull request by ArthurEncr**
# Description

Seedlot created feedback page created.

[Jira ticket 
](https://apps.nrs.gov.bc.ca/int/jira/browse/FSADT2-477)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have already been accepted and merged

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Frontend](https://nr-spar-webapp-46-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar-webapp/actions/workflows/merge-main.yml)

### CODE REVIEW

**Review by RMCampos — APPROVED** [review]
LGTM

File: src/views/SeedlotCreatedFeedback/index.tsx
Comment (mgaseta) [inline src/views/SeedlotCreatedFeedback/index.tsx:49]: Remove empty line.

File: src/views/SeedlotCreatedFeedback/styles.scss
Comment (mgaseta) [inline src/views/SeedlotCreatedFeedback/styles.scss:1]: For the whole page, add an space before the opening bracket on the selectors.

File: src/views/SeedlotCreatedFeedback/styles.scss
Comment (mgaseta) [inline src/views/SeedlotCreatedFeedback/styles.scss:20]: The color defined on the figma file is the primary button color (Light Theme/Button/$button-primary), adjust it here to reflect correctly the design.

File: src/views/SeedlotCreatedFeedback/styles.scss
Comment (mgaseta) [inline src/views/SeedlotCreatedFeedback/styles.scss:27]: Put an empty line in between rules.

File: src/views/SeedlotCreatedFeedback/styles.scss
Comment (mgaseta) [inline src/views/SeedlotCreatedFeedback/styles.scss:45]: Adjust tabs here.

File: src/views/SeedlotCreatedFeedback/styles.scss
Comment (mgaseta) [inline src/views/SeedlotCreatedFeedback/styles.scss:58]: Adjust tabs here.

File: src/views/SeedlotDashboard/styles.css
Comment (mgaseta) [inline src/views/SeedlotDashboard/styles.css:3]: Why change the color here?

**Review by mgaseta — CHANGES_REQUESTED** [review]
Good job 🚀✔! Just added a few markups, mostly related to code patterns, so no worries.

File: src/App.tsx
Comment (RMCampos) [reply src/App.tsx:54]: Great point! I would suggest `/seedlot/successfully-created`.

**Review by lucasbuenoencora — CHANGES_REQUESTED** [review]
LGTM! I guess that after the merge the new component is on the wrong directory, just need to put it inside the frontend folder.

**Review by lucasbuenoencora — APPROVED** [review]
Great Job :rocket: :heavy_check_mark:

**Pull request merged**
